tox.ini

```diff
@@ -49,4 +49,4 @@ max_line_length = 79
 ignore = E121,E122,E123,E125,E126,E127,E128,E226,E231,E251,E265,E266,E302,E402,E501,E712,E721,E731,E741,
          W291,W293,W391,W503,W504,
          F401,F403,F405,F841
-exclude = scipy/__config__.py
+exclude = scipy/__config__.py,scipy/_lib/highs
```

Comment:
HiGHS includes some Python files that were getting flagged by the linter -- we don't care about these!